experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -243,7 +243,7 @@ export const defaultSharedTreeOptions: Required<SharedTreeOptions> = {
  * @alpha
  */
 export class SharedTreeFactory implements IChannelFactory {
-	public type: string = "SharedTree";
+	public type: string = "https://graph.microsoft.com/types/tree";
```

Comment:
I'm not very opinionated on what we pick here. This is the style used for shared-string / matrix / pact-map, but I think the choices here come from aspirations to do more dynamic code loading than the types of scenarios we really aim to support these days.

Reply:
see also typed-tree, the other factory that wraps shared tree as that was by far the easiest way to get to our MVP API. I think that one matters more than this one, and also has the same issue.

Reply:
Where is that defined? Not finding references to it.

Reply:
nvm, needed to merge main. Fixed in latest.